### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Chloebert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,21 @@
-# Project 3 - Starter Kit - Symfony
+#Project aoyos - Starter Kit - Symfony
 
-## Presentation
+## Introduction
 
-This starter kit is here to easily start a repository for Wild Code School students.
+Hello fellow dev and welcome to the aoyos website project. 
+aoyos is an agence that coordinates events and is an intermediary between customers and Djs.
```
Comment (Ysedda) [inline README.md:6]: agency

File: README.md [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
 
-It's symfony website-skeleton project with some additional library (webpack, fixtures) and tools to validate code standards.
+First of all, before we begin, we will introduce the main points of our project. 
+
+The website is divided in three main parts :
```
Comment (Ysedda) [inline README.md:10]: into

File: README.md [hunk 3]
```diff
@@ -1,10 +1,21 @@
...
+
+The website is divided in three main parts : 
+
+-> A main website, which is mostly meant to introduce aoyos to the general public and has no particular feature aside from two forms : one for customers to submit their dj-ing requests, and one for djs to register.
+-> Once the djs are registered, they have a their own subsidiary website that they can use to modify their profiles, see all the events and position themselves on the ones they’re interested in. They can also upload their own invoices for the admin to see. 
+-> And finally, the admin part from which the admin can administrate all parts of their website.
+
+Now that this is out of the way, here are the technical aspects: 
+
+This project is based on a symfony website-skeleton project with some additional library (webpack, fixtures) and tools to validate code standards.
```
Comment (Ysedda) [inline README.md:18]: libraries

File: README.md [hunk 4]
```diff
@@ -97,6 +110,7 @@ Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c6
 ## Authors
 
 Wild Code School trainers team
+aoyos devs team
```
Comment (Ysedda) [inline README.md:113]: dev

File: README.md [hunk 5]
```diff
@@ -124,3 +138,6 @@ SOFTWARE.
 
 ## Acknowledgments
 
+Thank you to our trainers, Sylvain and Thibault for accompagnying us throughout this project.
```
Comment (Ysedda) [inline README.md:141]: Thanks to

File: README.md [hunk 6]
```diff
@@ -124,3 +138,6 @@ SOFTWARE.
 
 ## Acknowledgments
 
+Thank you to our trainers, Sylvain and Thibault for accompagnying us throughout this project. 
+Special thanks as well to our customers from aoyos, Gaëtan and Alice, for trusting us with this project. 
+And thank you to the Wild Code School for providing us with the base for this project.
```
Comment (Ysedda) [inline README.md:143]: thanks to

**Review by Ysedda — APPROVED** [review]

**Review by sblondeau — APPROVED** [review]

**Pull request merged**